### PR TITLE
Fix spelling and grammar errors

### PR DESCRIPTION
**Pull request by thejohnjansen**
Went through the spec and fixed some spelling and grammatical errors. I believe that I did not make any substantive changes. For those, I submitted bugs along the way.

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -576,10 +576,10 @@
 
   <section>
     <h3>Handling Errors</h3>
-    <p>Errors are represented in the WebDriver protocol with a HTTP response with a HTTP status in the 4xx or 5xx range and a JSON body containing details of the error. This JSON body has two fields; <code>error</code>, containing a string indicating the error type and <code>error</code> containing an implementation-defined string containing a human readable description of the kind of error that occured.</p>
+    <p>Errors are represented in the WebDriver protocol with a HTTP response with a HTTP status in the 4xx or 5xx range and a JSON body containing details of the error. This JSON body has two fields: <code>status</code>, containing a string indicating the error type and <code>error</code> containing an implementation-defined string containing a human readable description of the kind of error that occured.</p>
```
Comment (andreastt) [inline webdriver-spec.html]: This should be `error` for the string indicating the error type and `message` for the string with the human readable description.

(See the steps under **sending an error**.)

File: webdriver-spec.html [hunk 2]
```diff
@@ -1272,7 +1272,7 @@
 
           <li><p>If the task queued by that algorithm aborted
           before <a href="https://html.spec.whatwg.org/#traverse-the-history">traversing
-          the history</a>, return <a>success</a> with data null.
+          the history</a>, return <a>success</a> with data <var>null</var>.
```
Comment (andreastt) [inline webdriver-spec.html]: null’s not a variable but a value here.

File: webdriver-spec.html [hunk 3]
```diff
@@ -2232,9 +2230,7 @@ <h2 id="element-location-strategies">Element Location Strategies</h2>
    <h2 id=element-displayedness>Element Displayedness</h2>
 
    <p>The visibility of a <a href=http://w3c.github.io/dom/#element>Document element</a>
-    is guided by what is perceptually visible to the human eye.
-    An element's displayedness does in this context thus not relate
-    in any way to the
+    is guided by what is perceptually visible to the human eye. In this context, an element's displayedness does not relate to the
```
Comment (andreastt) [inline webdriver-spec.html]: Much better 😀

**Comment by AutomatedTester** [comment]
landed in 2d4c345 with fix ups

**Pull request closed without merging**